README.md

```diff
@@ -71,6 +71,7 @@ any C++ code.
   - [Business Logic Scripting](#business-logic-scripting-1)
   - [Preprocessing](#preprocessing)
   - [Decoupled Models](#decoupled-models)
+  - [AddSub in JAX](#addsub-in-jax)
```

Comment:
Move this immediately after AddSub numpy/pytorch examples

Reply:
Yes. Updated.